### PR TITLE
Add document message metadata

### PR DESCRIPTION
**Pull request by prakanth97**
## Purpose
$title

### CODE REVIEW

File: document-message/document-message.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+tagline: Transfer data to another application
+desc: The document message reliably transfers data from one application to another.
+helps: Ballerina supports a wide range of libraries for handling various messaging protocols, including REST, GraphQL, gRPC, and more. Additionally, it is equipped to handle a diverse array of MIME type attachments.
+tags: ["Document Message", "File Transfer", "Message Channnel", "Message Endpoint"]
```
Comment (manuranga) [inline document-message/document-message.yml]: [File Transfer](https://www.enterpriseintegrationpatterns.com/patterns/messaging/FileTransferIntegration.html) is not an EIP.

File: document-message/document-message.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+tagline: Transfer data to another application
+desc: The document message reliably transfers data from one application to another.
+helps: Ballerina supports a wide range of libraries for handling various messaging protocols, including REST, GraphQL, gRPC, and more. Additionally, it is equipped to handle a diverse array of MIME type attachments.
```
Comment (manuranga) [inline document-message/document-message.yml]: Do we have anything to say about "reliably" aspect. Ask from stdlib team.

File: document-message/document-message.yml (hunk 1 above)
Comment (manuranga) [reply document-message/document-message.yml]: In last review it was decided to remove non-eip tags.

File: document-message/document-message.yml (hunk 2 above)
Comment (manuranga) [reply document-message/document-message.yml]: I didn't meant to remove the word `reliably`, I mean to say, find if ballerina can do reliable messaging. If so mention how in the `helps` section.

Comment (prakanth97) [reply document-message/document-message.yml]: According to this https://lib.ballerina.io/ballerina/http/latest, client handles resilience in multiple way. Shall we include that? (client allows retry, timeout mechanism)

Comment (manuranga) [reply document-message/document-message.yml]: Yes. Something like "Depending on the protocol, different reliability mechanisms are availabl. example: Ballerina http retry feature."

**Review by manuranga — APPROVED** [review]

**Pull request merged**